### PR TITLE
Make 'skipInvalidResources' configurable via Helm chart + update README with missing config options

### PR DESCRIPTION
**Pull request by psiroky**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Small update to `logging-operator-logging` Helm chart, adding support for `skipInvalidResources`, which was recently added to `logging-operator` itself via https://github.com/banzaicloud/logging-operator/pull/1011.
It seems the update to this Helm chart was forgotten (or possibly skipped on purpose?)

### Why?
Users of the `logging-operator-logging` Helm chart can now easily set this configuration option / flag via `values.yaml` file.

### Checklist

- [X] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide) (as far as I can tell)
- [X] Related Helm chart(s) updated

### CODE REVIEW

File: charts/logging-operator-logging/values.yaml [hunk 1]
```diff
@@ -63,6 +63,9 @@ tls:
   fluentbitSecretName: ""
   fluentdSecretName: ""
 
+# Whether to skip invalid Flow and ClusterFlow resources
+skipInvalidResources: false
```
Comment (psiroky) [inline charts/logging-operator-logging/values.yaml:67]: Not sure about the position of this flag in file. The CRD (https://github.com/banzaicloud/logging-operator/blob/master/config/crd/bases/logging.banzaicloud.io_loggings.yaml) defines it right before `watchNamespaces`, so I figured I would do the same.

**Review by siliconbrain — DISMISSED** [review]
lgtm

**Comment by siliconbrain** [comment]
As for the documentation: yes, we're ~lazy~ spread too thin and can't keep everything up to date. Feel free to add any missing pieces, we'll try our best to review and merge asap 🙂

**Review by ahma — DISMISSED** [review]
LGTM Thanks @psiroky

**Comment by psiroky** [comment]
I pushed the README update. Please take a quick look. I basically followed the structure in values.yaml/template files and added those missing at the end of table.

**Review by ahma — APPROVED** [review]

**Pull request merged**